.setup/CONFIGURE_SUBMITTY.py

```diff
@@ -199,6 +200,7 @@ def __call__(self, parser, namespace, values, option_string=None):
     'course_code_requirements': "Please follow your school's convention for course code.",
     'sys_admin_email': '',
     'sys_admin_url': '',
+    'upgrade_linux_packages': False,
```

Comment:
Does this option require a migration to the existing installations of submitty's `config/submitty.json` file?

Reply:
Forgot that, I've added the migration.